Documentation/Blazorise.Docs/Pages/Blog/2022-06-09_ValidationWithDataAnnotations/Index.razor

```diff
@@ -1,6 +1,6 @@
 @page "/blog/blazor-form-validation-with-data-annotations"
 
-<Seo Canonical="/blog/blazor-form-validation-with-data-annotations" Title="Blazorise Form Validation With Data Annotations" Description="Learn all about how to utilize Blazorise form validation to quickly validate your input fields without the model instance." ImageUrl="img/blog/2022-06-09/Blazorise-Form-Validation-With-Data-Annotations.png" />
```

Comment:
I think there was a merge issue here. An `ImageUrl` should stay in Seo.

Reply:
I don't quite get it, I don't think I did any manual merge? Anyway I'll re-add it.

Reply:
It did the same for me yesterday. Not sure why.